global/scripts/MyelinMap_BC.sh

```diff
@@ -0,0 +1,180 @@
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"         # Debugging functions; also sources log.shlib
+g_matlab_default_mode=1
+defaultSigma=$(echo "sqrt(200)" | bc -l)
+
+#description to use in usage - syntax of parameters is now explained automatically
+opts_SetScriptDescription "implements CIFTI-based Myelin map bias corrections from one resolution to another; resolution includes native, 164k, 32k"
+
+#mandatory
+#general inputs
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder that contains all subjects"
+opts_AddMandatory '--subject' 'Subject' '100206' "one subject ID"
+opts_AddMandatory '--registration-name' 'RegName' 'MSMAll' "the registration string corresponding to the input files, e.g. 'MSMAll' or ''"
+opts_AddMandatory '--input-mesh' 'InputMesh' 'meshnum or native' "input resolution mesh node count (in thousands) or 'native', like '164' for 164k_fs_LR"
+opts_AddMandatory '--output-mesh' 'OutputMesh' 'meshnum or native' "low resolution mesh node count (in thousands) or 'native', like '32' for 32k_fs_LR"
+#optional inputs
+opts_AddOptional '--use-ind-mean' 'UseIndMean' 'YES or NO' "whether to use the mean of the subject's myelin map as reference map's myelin map mean , defaults to 'NO'" 'NO'
+opts_AddOptional '--mcsigma' 'CorrectionSigma' 'number' "myelin map bias correction sigma, default '$defaultSigma'" "$defaultSigma"
+opts_AddOptional '--matlab-run-mode' 'MatlabRunMode' '0, 1, or 2' "defaults to $g_matlab_default_mode
+0 = compiled MATLAB
+1 = interpreted MATLAB
+2 = Octave" "$g_matlab_default_mode"
+opts_ParseArguments "$@"
+
+if ((pipedirguessed))
+then
+    log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+# Verify required environment variables are set and log value
+log_Check_Env_Var HCPPIPEDIR
+log_Check_Env_Var CARET7DIR
+
+#display HCP Pipeline version
+log_Msg "Showing HCP Pipelines version"
+"${HCPPIPEDIR}"/show_version --short
+
+#display the parsed/default values
+opts_ShowValues
+
+# ------------------------------------------------------------------------------
+#  Main processing of script.
+# ------------------------------------------------------------------------------
+log_Msg "Starting main functionality"
+Caret7_Command=${CARET7DIR}/wb_command
+RegNameToUse=""
+if [[ "$RegName" != "" ]]; then
+	RegNameToUse=_${RegName}
+fi
+
+# default folders
+SubjFolder=${StudyFolder}/${Subject}
+AtlasSpaceFolder=${SubjFolder}/MNINonLinear
+T1wFolder=${SubjFolder}/T1w
+#NativeT1wFolder=${T1wFolder}/Native
+#NativeFolder=${AtlasSpaceFolder}/Native
+#DownSampleFolder=${AtlasSpaceFolder}/fsaverage_LR${LowResMesh}k
+#DownSampleT1wFolder=${T1wFolder}/fsaverage_LR${LowResMesh}k
+
+# convert upper case to lower case
+InputMesh=`echo "$InputMesh" | tr '[:upper:]' '[:lower:]'`
+OutputMesh=`echo "$OutputMesh" | tr '[:upper:]' '[:lower:]'`
```

Comment:
I don't think we need to support people giving "NaTiVe" in either of these arguments.